### DISCUSSION TITLE
part-ii/2015-77/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#210] [Question comments]
Part II 2015 Logic and Set Theory: Paper 4, Section II, I 

https://questions.tripos.org/part-ii/2015-77/

**Comment by ArthurConmy** [comment]
I get 

<details>
<summary> i) </summary>

$\omega + 4$
</details>

<details>
<summary> ii) </summary>

$\omega $
</details>

<details>
<summary> iii) </summary>

$\omega $
</details>
